### PR TITLE
[bugfix] keys returns nodes, not sexpressions

### PR DESCRIPTION
**Pull request by witek**

### CODE REVIEW

File: src/borkdude/rewrite_edn/impl.cljc [hunk 1]
```diff
@@ -241,7 +241,7 @@
            ks '()]
         (if (z/rightmost? zloc)
           ks
-          (let [k (z/sexpr zloc)]
+          (let [k (first zloc)]
```
Comment (borkdude) [inline src/borkdude/rewrite_edn/impl.cljc:244]: I think calling `z/node` is better here.

**Comment by borkdude** [comment]
why did you close? :)

**Comment by witek** [comment]
I am new to Github Pull Requests. I thought, I have to create a new one.

**Comment by borkdude** [comment]
No, you can just push to your branch and the PR is updated :)

**Pull request closed without merging**